Packs/GmailSingleUser/Integrations/GmailSingleUser/GmailSingleUser_description.md

```diff
@@ -1,14 +1,62 @@
+3. Enter the following Client ID: `391797357217-pa6jda1554dbmlt3hbji2bivphl0j616.apps.googleusercontent.com`
+   You see the `Demisto App` in the results page. 
+  ![Demisto App](doc_imgs/demisto-app-result.png)
+4. Select the app and grant the app access as `Trusted`.
+5. Add the Demisto app client ID `391797357217-pa6jda1554dbmlt3hbji2bivphl0j616.apps.googleusercontent.com` to the integration configuration.
+6. Proceed to 'Authorization Flow In Cortex XSOAR' to configure OAuth 2.0 authorization in Cortex XSOAR.
+
+
+### All Account Types:
+
+If you are not a GSuite Admin, you must use your own Google app instead of the default Demisto app.
+1. Go to the [developers credentials page](https://console.developers.google.com/apis/credentials) (you may need to set up a [new project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) if you haven't already).
+2. If needed, configure the [Consent Screen](https://developers.google.com/workspace/guides/configure-oauth-consent). Fill in the Consent Screen information you want to display to your users.
+3. Make sure in the consent screen that you publish the app by clicking `Publish App` and confirming.
+  ![OAuth-Consent-Screen-Publication](doc_imgs/publish-OAuth-consent-screen.png)
+4. In the credentials page choose: `Create Credentials` -> `OAuth client ID`.
+  ![Create Credentials](doc_imgs/create-credentials.png)
+5. When creating the OAuth client ID, select **iOS** as the type (this type allows apps to work only with a client ID). **iOS** is the type used for all apps which are not Android (including desktop types).
+6. Name the app and bundle. You can choose a dummy bundle ID such as `com.demisto.app`.
+  ![OAuth App](doc_imgs/oauth-app.png)
+7. Make sure to [enable the Gmail API](https://console.developers.google.com/apis/api/gmail.googleapis.com/overview) if you haven't already.
+8. After you create the app, copy the *client id* of the app that you created to the integration configuration.
+9. Proceed to 'Authorization Flow In Cortex XSOAR' to configure OAuth 2.0 authorization in Cortex XSOAR.
+
+### Authorization Flow In Cortex XSOAR
+1. Create and save an integration instance of the Gmail Single User integration. Do not fill in the *Auth Code* field, this is obtained in the next steps.
+2. To obtain the **Auth Code**, run the following command in the playground: ***!gmail-auth-link***. Access the link you receive to authenticate your Gmail account.
+3. If you get a message from Google saying that it cannot verify the application, click **proceed** and click enter for the app name to give the app you created permissions to your Google account. Then click **proceed**.
+4. Complete the authentication process and copy the received code to the **Auth Code** configuration parameter of the integration instance. 
+5. Save the instance.
+6. To verify that authentication was configured correctly, run the ***!gmail-auth-test***.
+
+### Fetched Incidents Data
```

Comment:
Remove this, not necessary in the description file

Reply:
ack